app/graphql/publishing_api_schema.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class PublishingApiSchema < GraphQL::Schema
+  query(Types::QueryType)
+
+  # For batch-loading (see https://graphql-ruby.org/dataloader/overview.html)
+  use GraphQL::Dataloader
+
+  # GraphQL-Ruby calls this when something goes wrong while running a query:
+  def self.type_error(err, context) # rubocop:disable Lint/UselessMethodDefinition
+    # if err.is_a?(GraphQL::InvalidNullError)
+    #   # report to your bug tracker here
+    #   return nil
+    # end
+    super
+  end
+
+  # Union and Interface Resolution
+  def self.resolve_type(_abstract_type, _obj, _ctx)
+    # TODO: Implement this method
+    # to return the correct GraphQL object type for `obj`
```

Comment:
Note this outstanding TODO

Reply:
I'm assuming this will get removed in a later PR when we put some things in this method?

Reply:
Yeah I think so!